model/src/main/java/com/basistech/rosette/dm/internal/TextWrapper.java

```diff
@@ -40,6 +40,7 @@
  * on token indexes.  The underlying <code>Text</code> object contains only
  * character offsets.
  */
+@SuppressWarnings("deprecation")
```

Comment:
@benson-basis - I cannot find a TextWrapperText class in this repo.  Could you
post a git link to it?

The TextWrapperTest in annotated-data-model-internal does use ARA only because
that's what we had when the test was developed.  It's really testing the functionality
of TextWrapper.  We just convert ARA to ADM before each test.  We could easily
change that now.